content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -101,32 +101,34 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-### Latency for Istio 1.21
+### Latency for Istio 1.22
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio 1.21 (i.e. Istio with telemetry v2),
-the two proxies add about 0.182 ms and 0.248 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+proxy. In the default (i.e. ambient mode) configuration of Istio 1.22,
+the two proxies add about 0.17 ms and 0.20 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency. Waypoint is better than sidecar.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
-for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.
+for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 1,2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.
 
 Note: This testing was performed on the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster). Different hardware will give different values.
 
 <br><img width="90%" style="display: block; margin: auto;"
-    src="istio-1.21.0-nighthawk-90.png"
+    src="istio-1.22.0-fortio-90.png"
     alt="P90 latency vs client connections"
     caption="P90 latency vs client connections"
 />
 <p><h2 style="text-align: center;"> P90 latency vs client connections </h2></p><br>
```

Comment:
Why is the header below the graph? I would expect either something that is small (not h2) or for it to be above

Reply:
agreed, i got confused a bit on this too